install

```diff
@@ -243,18 +244,24 @@ Dir.chdir HOMEBREW_PREFIX do
     system(*args)
 
     system git, "reset", "--hard", "origin/master"
+
+    # no analytics during installation
+    ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
```

Comment:
Maybe just worth setting this super early in this file so if other `brew` commands are added outside of this `if` then they won't have analytics run.